packages/shared/src/utils/os/os_detect.ts

```diff
@@ -82,3 +82,19 @@ export const mobileOSDetect = () => {
 
     return 'unknown';
 };
+
+export const checkUserBrowser = () => {
```

Comment:
Can we move this function to a new file? May be browser detect?

Reply:
When I was creating suggested folder I found that it already exist and contains similar function for Safari detection)))) Will remove my implementation and ad existing one

Reply:
Oh, unfortunately existing function `isSafari()` from _packages/shared/src/utils/browser/browser_detect.ts_ is not working on mobile, so I'll add mine in that file too